templates/dashboard/company/modal/confirm_promote.html

```diff
@@ -0,0 +1,25 @@
+{% extends "dashboard/base_modal.html" %}
+
+{% load i18n %}
+
+{% block form_class %}{% endblock %}
+
+{% block action_url %}
+{% url 'dashboard:customer-promote' pk=customer.pk %}
```

Comment:
I'm not sure I understand the purpose of this view - it's copied from the customer, but is it valid for companies as well?

Reply:
This was a result of me just copying the customer folder and not realizing that there was a template in there I wasn't using. I'll remove it. I also checked to make sure it wasn't being referenced in any of the company files.